amethyst_network/Cargo.toml

```diff
@@ -30,6 +30,6 @@ bincode = "1.0"
 log = "0.4.6"
 uuid = { version = "0.7.1", features = ["v4","serde"] }
 thread_profiler = { version = "0.3" , optional = true }
-laminar = "0.2"
+laminar = { path = "../../laminar" }
```

Comment:
adding comment here so we dont forget to merge with this in 😄

Reply:
I also left that comment in the `Notice`

Reply:
done